nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Deprecated Distributor in NServiceBus Version 6
```

Comment:
I'd prefer `Migrating the Distributor to use Sender Side Distribution in NSB Version 6` but do we want to write "NSB"? (also it seems to be very long). Do we need to state versions,since it's part of the NServiceBus6 upgrade section?

e.g. how about "Migrating the Distributor to use Sender Side Distribution" ?